app/router.go

```diff
@@ -104,7 +104,7 @@ var topologyRegistry = map[string]topologyView{
 	"applications": {
 		human:    "Applications",
 		parent:   "",
-		renderer: render.FilterUnconnected{Renderer: render.ProcessRenderer},
+		renderer: render.FilterUnconnected{Renderer: render.ProcessWithContainerNameRenderer{}},
```

Comment:
The process renderer is used by the container render; there very little point in going to all the effort of labelling up the processes if we're just going to blow it away in the container renderer.